src/libs/API.js

```diff
@@ -4,6 +4,19 @@ import * as Request from './Request';
 import * as SequentialQueue from './Network/SequentialQueue';
 import {version} from '../../package.json';
 
+/**
+ * All calls to API.write() will be persisted to disk as JSON with the params, successData, and failureData.
+ * This is so that if the network is unavailable or the app is closed, we can send the WRITE request later.
+ *
+ * @param {String} command - Name of API command to call
+ * @param {Object} apiCommandParameters - Name value pairs of parameters to send to the API
```

Comment:
```suggestion
 * @param {Object} apiCommandParameters - The object of parameters to send to the API
```
You have it like this for Api.read.




"Name value pairs" confused me because of our nameValuePairs 😆.

Reply:
> You have it like this for Api.read.

Good catch, I'm going to update it to make it consistent on all the functions

> "Name value pairs" confused me because of our nameValuePairs 😆.

Hehe it was originally `The object` https://github.com/Expensify/App/pull/9558#discussion_r906173951 😄 but I think with `@param {Object}` that can't be confusing when the function is used (vscode and/or eslint will complain if an object is not used)

Reply:
But yeah after reading it again I got the point, for someone new to the codebase it can be confusing wondering if this object should contain or is related to `nameValuePairs`, @tgolen do you think it's ok to revert it back to `The object`?

Reply:
Atm I revert it back to `The object`, I think @aldo-expensify is right, and imo it's better to be obvious than create a slight confusion

Reply:
In fact, you could just say "Parameters to send to the API", there is already the {Object} saying that it is an object and "The object" doesn't add much

Reply:
I was just about to say that, Aldo! Great minds.

Reply:
I like it! updating it